src/components/Frame/types.ts

```diff
@@ -51,6 +51,13 @@ export interface ContextualSaveBarProps {
 
 // Toast
 
+export enum ToastDuration {
+  Fast = 3000,
+  Base = 5000,
```

Comment:
How do we feel about using `base` as a value? We use it a lot for our sass functions, but it's a default. So we would use `spacing()` rather than `spacing(base)`

Reply:
I'm not clear on how a px value relates to a ms value?

Reply:
It doesn't I was just using it for comparison. Since we would be using `duration="base"`, which I don't find very descriptive.

Reply:
Ohhh! When you asked how we feel about base, I thought you were suggesting to use it and was quite confused. Now I understand what you're saying.

Would it make more sense to call this `Default`? Or?

Reply:
What about:

```
Fast = 3000
Default = 5000
Slow = 7000
Slower = 10000
Slowest = 15000
```

Reply:
```tsx
enum Duration {
  Fastest = 1000
  Fast = 3000
  Default = 5000
  Slow = 7000
  Slower = 9000
}
```

Feels pretty natural to me, would love to hear others thoughts on it. Since if we ever implement another component with a duration prop we'll *probably* want to reuse this enum

Reply:
We increased the default duration for accessibility reasons from 3000 to 5000. Some toasts exist in web with 3000 as a duration because it used to be the default. But I don't know why we would go any less than 3000. 1000ms is too fast for many people to actually see it and would have accessibility issues.

We need 10000 for the action prop: https://github.com/Shopify/polaris-react/pull/919/files#diff-f6f39e05b67f0498645df16bf2a7c4eaR460